src/components/Thumbnail/Thumbnail.js

```diff
@@ -138,10 +152,15 @@ class Thumbnail extends React.PureComponent {
             width: THUMBNAIL_SIZE,
             height: THUMBNAIL_SIZE,
           }}
-          ref={this.thumbContainer}
-          onDragStart={this.onDragStart}
-          draggable={isDraggable}
-        />
+        >
+          <div
+            className="thumbnail"
```

Comment:
Do we have to have this `.thumbnail` div? For the checkmark can we just make it positioned relative to the canvas element?

Reply:
I updated it to add the check mark on the first child (in case it's a XOD file and the first element is an img).